projects/i18n/switch/load-lang.ts

```diff
@@ -0,0 +1,33 @@
+import {TuiLanguage, TuiLanguageName} from '@taiga-ui/i18n/interfaces';
+import {from, Observable, of} from 'rxjs';
+import {map} from 'rxjs/operators';
+
+import {TuiLangChunkLoader} from './options';
+
+interface Options {
+    storage: Storage;
+    key: string;
+    fallback: TuiLanguage;
+    loader: TuiLangChunkLoader | null;
+}
+
+export function tuiAsyncLoadLanguage(options: Options): Observable<TuiLanguage> {
+    const {storage, key, fallback, loader} = options;
+    const language = storage.getItem(key) as TuiLanguageName | null;
```

Comment:
I think it's a bit too much to pass storage and key in this utility function. Let's just pass `language`.

Reply:
fixed